db/schema.rb

```diff
@@ -32,6 +32,18 @@
     t.index ["town"], name: "index_address_base_on_town"
   end
 
+  create_table "address_base_tmp", primary_key: "uprn", id: :string, force: :cascade do |t|
```

Comment:
I don't think this table exists in deployed environments?  (it's not in integration anyway) - maybe do a `git checkout --  db/schema.rb` to revert this change

Reply:
This gets spun up and then removed once the addressbase update process is complete.